api/v1alpha1/service_fence.proto

```diff
@@ -4,7 +4,7 @@
 */
 
 syntax = "proto3";
-package slime.microservice.v1alpha1;
+package slime.microservice.lazyload.v1alpha1;
 
 option go_package = "slime.io/slime/pkg/apis/microservice/v1alpha1";
```

Comment:
wrong go package

Reply:
change to go_package = "slime.io/slime/modules/lazyload/api/v1alpha1";